kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -133,7 +136,7 @@ public synchronized List<String> getInterfaces() throws DBusException {
         return supportedDeviceNames;
     }
 
-    public synchronized NetInterface<NetInterfaceAddress> getInterfaceStatus(String interfaceName)
+    public synchronized NetworkInterfaceStatus getInterfaceStatus(String interfaceName, NetworkService networkService)
```

Comment:
I'm not entirely sure we need to add the dependency to the `NetworkService`. The "[Udi](https://networkmanager.dev/docs/api/latest/gdbus-org.freedesktop.NetworkManager.Device.html#gdbus-property-org-freedesktop-NetworkManager-Device.Udi)" and "[Path](https://networkmanager.dev/docs/api/latest/gdbus-org.freedesktop.NetworkManager.Device.html#gdbus-property-org-freedesktop-NetworkManager-Device.Path)" property could be a way to obtain the same informations.

For example on a RPi I get:

![image](https://user-images.githubusercontent.com/22748355/220158898-9185dbe8-d4a0-4964-9bac-721c542e4775.png)

Reply:
The property that we need to show in the webUi is the usb path in the form of connection point (see [here](https://stackoverflow.com/questions/33140787/determine-usb-device-file-path) for details). 
So, Udi and Path are not what we need. But we can discuss about if the really need that informations...